app/src/main/java/com/jp/wasabeef/glide/transformations/internal/Utils.java

```diff
@@ -1,38 +0,0 @@
-package com.jp.wasabeef.glide.transformations.internal;
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.